appinventor/components/src/com/google/appinventor/components/runtime/AppInventorCompatActivity.java

```
@@ -85,6 +90,9 @@ public void onCreate(Bundle icicle) {
 
     super.onCreate(icicle);
 
+    relativeLayout = new RelativeLayout(this);
+    floatingActionButton = new FloatingActionButton(this);
+    floatingActionButton.hide();
```

Why is it hidden by default? Make sure to leave a comment if it's necessary to hide